docs/sources/administration/correlations/use-variables-and-transformations/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Use variables and transformations in a correlation
+weight: 60
+---
+
+# Use variables and transformations in a correlation
+
+## Before you begin
+
+This example walks through creating a link in a test data source but the same principles apply to any data source.
+
+The example emulates a scenario with two data sources:
+
+- Logs containing lines in the format: “2020-01-01 10:00 level=error message=error service=app1.loginService” stored in a field named “msg”
+- Metrics for application included in the service name of a log line (e.g. app1)
+
+Instructions below show how to set up a link that can run metrics query for the host included in each log line with provisioning and regex transformation. Additionally, a link with a query containing the full name of the service is set up to demonstrate the logfmt transformation.
+
+## Use variables and transformations in provisioning
+
+1. Add the following provisioning configuration to your Grafana
+
+   ```yaml
+   datasources:
+     - name: Target
+       uid: test-target
+       type: testdata
+
+     - name: Source
+       uid: test-source
+       type: testdata
+     - name: Source
+       uid: test-source
+       type: testdata
+       correlations:
+         - targetUID: test-target
+           label: App metrics
+           description: Application HTTP request metrics
+           config:
+             type: query
+             target:
+               scenario_id: random_walk
+               alias: $${application}
+             field: msg
+             transformations:
+               - type: regex
+                 field: msg
+                 expression: service=(\w+)\.\w+
+                 mapValue: application
+         - targetUID: test-target
+           label: Service metrics
+           description: Service metrics
+           config:
+             type: query
+             target:
+               scenario_id: random_walk
+               alias: $${service}
+             field: msg
+             transformations:
+               - type: logfmt
+                 field: msg
+   ```
+
+   Two data sources are created: Source (emulating logs data source) and Target (emulating metrics data source):
+
+   - A correlation called “App metrics” is created targeting the Target data source with its UID.
```

Comment:
same: https://grafana.com/docs/writers-toolkit/style-guide/style-conventions/#unordered-lists

Reply:
Updated in 89cf03c6a6c336ad6479e2d4d51c2c0adf3a78d0.